app/models/account.rb

```diff
@@ -58,6 +58,7 @@ def self.columns
   include CreditCard
   include Gateway
   include States
+  require_dependency 'account/domains'
```

Comment:
I think this is not necessary 🤔 Have you tried if it works without this line? 🤔

Reply:
Yes :) It will trigger rails autoload and load `::Domains` instead of `Account::Domains`. Even full  `::Account::Domains` does not work.